sunpy/io/srs.py

```diff
@@ -0,0 +1,124 @@
+"""
+This module implements SRS File Reader.
+"""
+__author__ = "Sudarshan Konge"
+__email__ = "sudk1896@gmail.com"
+
+from astropy.table import Table, Column, vstack
+import collections, calendar, re
+
+__all__ = ['read']
+
+def read(filepath):
+    """
+    Method for reading an SRS File.
+    """
+    File = open(filepath, 'r')
+    lines = list()
+    for line in File:
+        arr = line.split()
+        store = list()
+        for i in range(0, min(len(arr), 8)):
+            store.append(arr[i])
+        lines.append(store)
+
+    #Store table meta-data. Only first two lines
+    #of SRS text file store the meta-data.
+    meta_data = collections.OrderedDict()
+    regex = re.compile('[^a-zA-Z]')
+    try:
+        meta_data[regex.sub('', lines[0][0])] = lines[0][1]
+        meta_data[regex.sub('', lines[1][0])] = ' '.join(lines[1][1:])
+    except IndexError:
+        print ("SRS meta-data is not available.")
+
+    #Problem: An array of strings. We need to extract
+    #three tables from these I, IA and II.
+
+    table = list() #This table holds the three tables which we would later merge.
+    indices = list()
+    
+    for i in range(0, len(lines)):
+        if (lines[i][0][0] == 'I'):
+            indices.append(i)
+    indices.append(len(lines))
+    indices.sort()
+    
+    for i in range(0, len(indices) - 1):
+        cols = lines[indices[i]+1]
+        temp_table = Table(names = cols, dtype=['object_']*len(cols))
+        #If a table is empty, we are not adding it.
```

Comment:
there's a problem here... I rather have a `None` row than not at all. The thing is that it's not the same that data is missing than there's not data. So if one day they don't measure because it's cloudy that doesn't mean that there's not an active region, but if they do measure and they find nothing it's 0 not not measured. So, I prefer to put `None` if the row exists but nothing if they don't.